setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='gramhopper',
-    version='1.0.6',
+    version='1.0.7',
```

Comment:
Can't the version number be defined once somewhere?

Reply:
I hope it can. I'll find a more convenient way to handle it later on.

Reply:
KK